draft-wood-ohai-unreliable-ohttp.md

```diff
@@ -183,16 +183,21 @@ A Client interacts with the Oblivious Relay Resource by constructing an
 Encapsulated Request as described in {{OHTTP}}. This Encapsulated Request
 is included as the content of a POST request to the Oblivious Relay Resource.
 Importantly, this request MUST include the "message/ohttp-ack" Media Type
-in the Accept header (see {{iana-ack}}). The Client receives a 202 Accepted
-response with content type "message/ohttp-ack" and empty body upon successful
-transmission of the request. Any other response is considered invalid.
+in the Accept header (see {{iana-ack}}). The Client receives a `202 Accepted`
+response with content type "message/ohttp-ack" and an empty body upon successful
+transmission of the request. If the Accept header also allows the normal
+"message/bhttp" content type, the client has left the choice of reliable or
+unreliable delivery up to the Relay and Gateway Resources, which may
+return "200 OK" along with an Encapsulated Body instead.
+Any other response is considered invalid.
 
 Upon receipt of an unreliable OHTTP request from the Client, the Oblivious
-Relay Resource MUST reply with a 202 Accepted response with the "message/ohttp-ack"
-content type to the Client and buffer the request to be sent to the Oblivious
-Gateway Resource at some point in the future. Similarly, upon receipt of an
+Relay Resource MUST reply with a `202 Accepted` response with the "message/ohttp-ack"
+content type to the Client. It SHOULD buffer the request to be sent to the Oblivious
+Gateway Resource at some point in the future, but MAY chose to forward it immediately.
```

Comment:
```suggestion
content type to the Client. It can buffer the request to be sent to the Oblivious
Gateway Resource at some point in the future, or choose to forward it immediately.
```

I don't think we should require buffering here, since that's an implementation-specific decision.

Reply:
Ok